apps/marketplace/components/SellerSelect/SellerSelect.js

```diff
@@ -0,0 +1,104 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import AUtextInput from '@gov.au/text-inputs/lib/js/react.js'
+import AUbutton from '@gov.au/buttons/lib/js/react.js'
+import findSuppliers from 'marketplace/actions/supplierActions'
+import styles from './SellerSelect.scss'
+
+export class SellerSelect extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      sellers: [],
+      inputValue: ''
+    }
+    this.handleOnChange = this.handleOnChange.bind(this)
+    this.handleSellerSelectClick = this.handleSellerSelectClick.bind(this)
+    this.handleSearchClick = this.handleSearchClick.bind(this)
+  }
+
+  handleOnChange(e) {
+    const keyword = e.target.value
+    this.setState({
+      inputValue: keyword
+    })
+
+    if (keyword && keyword.length > 2) {
+      findSuppliers(keyword)
+        .then(data => {
```

Comment:
`dmapi()` and its response returns a `Promise` so we can use `then()` etc directly in the component rather than use redux's dispatch system to handle responses and state changes.